test/wpt/status/dom/events.json

```
@@ -0,0 +1,40 @@
+{
+  "Event-dispatch-listener-order.window.js": { "fail": { "unexpected": [ "document is not defined" ] } },
```

nits. It is OK not to fix this.
```suggestion
  "Event-dispatch-listener-order.window.js": { "fail": "document is not defined" },
```